target/scripts/startup/setup-stack.sh

```diff
@@ -23,7 +23,7 @@ function _setup_supervisor
         exit
         ;;
 
-      'warn' )
+      ( 'warn' )
```

Comment:
I'm not able to highlight the line below with the `* )` case, but I assume you want that to be consistent and have a `(` prefix too?

Reply:
Done in d2c9aeda3725e6963105d845520d3a2e07a7aeac